src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/models/_container_service_client_enums.py

```diff
@@ -1169,12 +1169,12 @@ class WorkloadRuntime(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     """Nodes will use Kubelet to run standard OCI container workloads."""
     WASM_WASI = "WasmWasi"
     """Nodes will use Krustlet to run WASM workloads using the WASI provider (Preview)."""
-    KATA_MSHV_VM_ISOLATION = "KataMshvVmIsolation"
+    KATA_VM_ISOLATION = "KataVmIsolation"
```

Comment:
no need to update the SDK, it's auto generated

Reply:
Removed change.